doc/man3/SSL_CONF_cmd.pod

```diff
@@ -85,6 +85,15 @@ its preference list. This usually indicates a client without AES hardware
 acceleration (e.g. mobile) is in use. Equivalent to B<SSL_OP_PRIORITIZE_CHACHA>.
 Only used by servers. Requires B<-serverpref>.
 
+=item B<-legacy_ssl_eof>
+
+Some TLS implementations do not send close_notify on shutdown. Before OpenSSL
+3.0 the SSL_ERROR_SYSCALL with errno=0 was reported in such cases. This
+behaviour cotradicts the B<SSL_ERROR_SYSCALL> documentation. Since OpenSSL 3.0
+the returned error is B<SSL_ERROR_SSL> with a meaningful error on the error
+stack. This option allows to get the legacy error in such cases. Equivalent to
```

Comment:
This option gives the legacy error in such cases.

Reply:
Fixed. Thanks!